### DISCUSSION TITLE
Will posit.cloud evolve to Positron vs RStudio?

### DISCUSSION THREAD

**Discussion by critmcdonald** [posit-dev/positron#4125] [Q&A] [answered]
I'm curious about how the development of Positron might influence the hosted posit.cloud platform. Could future posit.cloud users have the choice of using Positron over RStudio?

**Comment by juliasilge** [comment, marked as answer]
We are definitely exploring whether folks are interested in Positron being available at posit.cloud! Currently, we only have Positron available as a desktop app, but as we move forward with hosted environments, we can evaluate whether posit.cloud will be one of the ways the Positron is available to folks. We know this is especially relevant for educational use.
